<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screneShot_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>screenshot:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="task_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>201</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Task:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="screenShot_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="user_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>201</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="task_CB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="user_CB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="description_LE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="description_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>201</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLog_L">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>201</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TimeLog: (minutes)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timeLog_LE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>description_LE</tabstop>
  <tabstop>timeLog_LE</tabstop>
  <tabstop>screenShot_B</tabstop>
  <tabstop>user_CB</tabstop>
  <tabstop>task_CB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
